File: ios/BockMedia/Domain/HomeDecadeSections.swift
import Foundation

struct HomeDecadeSection: Equatable {
    let id: String
    let title: String
}

/// Fixed home rows — one section per decade, all matching library playlists.
enum HomeDecadeSections {
    static let sixties = HomeDecadeSection(id: "60s", title: "60s")
    static let seventies = HomeDecadeSection(id: "70s", title: "70s")
    static let eighties = HomeDecadeSection(id: "80s", title: "80s")
    static let nineties = HomeDecadeSection(id: "90s", title: "90s")

    static func all() -> [HomeDecadeSection] {
        [sixties, seventies, eighties, nineties]
    }
}

File: android/app/src/main/kotlin/com/bockmedia/console/domain/model/HomeDecadeSections.kt
package com.bockmedia.console.domain.model

data class HomeDecadeSection(
    val id: String,
    val title: String,
)

/** Fixed home rows — one section per decade, all matching library playlists. */
object HomeDecadeSections {
    val sixties = HomeDecadeSection("60s", "60s")
    val seventies = HomeDecadeSection("70s", "70s")
    val eighties = HomeDecadeSection("80s", "80s")
    val nineties = HomeDecadeSection("90s", "90s")

    fun all(): List<HomeDecadeSection> = listOf(sixties, seventies, eighties, nineties)
}
